tests/baselines/reference/accessorAccidentalCallDiagnostic.errors.txt

```diff
@@ -0,0 +1,18 @@
+tests/cases/compiler/accessorAccidentalCallDiagnostic.ts(6,14): error TS2349: This expression is not callable.
+  Type 'Number' has no call signatures.
+
+
+==== tests/cases/compiler/accessorAccidentalCallDiagnostic.ts (1 errors) ====
+    // https://github.com/microsoft/TypeScript/issues/24554
+    class Test24554 {
+        get property(): number { return 1; }
+    }
+    function test24554(x: Test24554) {
+        return x.property();
+                 ~~~~~~~~
+!!! error TS2349: This expression is not callable.

```

Comment:
I would much rather replace this message instead of tacking on related info.

If it's too hard to replace the message, it should be an elaboration, not related info. Elaborations provide additional messages; related info is designed to point to a different location.